packages/node/test/handlers.test.ts

```diff
@@ -0,0 +1,73 @@
+import { parseRequest } from '../src/handlers';
+import { Event } from '../src';
+
+describe('parseRequest', () => {
+  const mockReq = {
+    method: 'GET',
+    url: '/some/path?key=value',
+    headers: {
+      host: 'mattrobenolt.com',
+    },
+    cookies: { test: 'test' },
+    body: '',
+    user: {
+      id: 123,
+      username: 'tobias',
+      email: 'tobias@mail.com',
+      custom_property: 'foo',
+    },
+  };
+
+  describe('parseRequest.user properties', () => {
```

Comment:
Thought i'd add a bit of test coverage for some of this logic for both the `user` and `request` properties